### PR TITLE
Attempt clearer language for fetch and resources

### PR DESCRIPTION
**Pull request by Amorymeltzer**
It was never particularly clear to me what "resources" meant, in part because I think it was used over-broadly.  Maybe I'm the outlier, but [two years later](https://github.com/caskroom/homebrew-cask/pull/2970) I'm still confused.

Happy to entertain alternative language choices here, I'm not wed to these.

### CODE REVIEW

File: USAGE.md [hunk 1]
```diff
@@ -67,12 +67,12 @@ Note that `uninstall --force` is currently imperfect. See the man page for more
 
 * `info` — displays information about the given Cask
 * `list` — with no args, lists installed Casks; given installed Casks, lists staged files
-* `fetch` — downloads Cask resources to local cache (with `--force`, re-download even if already cached)
+* `fetch` — downloads files containing Cask applications to local cache (with `--force`, re-download even if already cached)
```
Comment (vitorgalvao) [inline USAGE.md]: “files containing Cask applications” is confusing to me.

“downloads files need to install applications to local cache (with `--force`, re-download even if already cached)”?

File: doc/src/brew-cask.1.md [hunk 2]
```diff
@@ -123,7 +124,7 @@ names, and other aspects of this manual are still subject to change.
     files. `zap` actions are variable, depending on the level of detail
     defined by the Cask author.
 
-    **`zap` may remove resources which are shared between applications.**
+    **`zap` may remove files which are shared between applications.**
```
Comment (vitorgalvao) [inline doc/src/brew-cask.1.md]: It’s not just files, though, as `zap` can do everything `uninstall` can.

File: USAGE.md (hunk 1 above)
Comment (vitorgalvao) [inline USAGE.md]: Better yet, why not just steal directly from the manpage? “download remote application files for the given Cask to the local cache (with `--force`, re-download even if already cached)” seems pretty clear to me.

Comment (Amorymeltzer) [inline USAGE.md]: Hmm, I like that latter suggestion a lot.

File: USAGE.md [hunk 3]
```diff
@@ -67,12 +67,12 @@ Note that `uninstall --force` is currently imperfect. See the man page for more
...
 * `doctor` — checks for configuration issues
 * `cleanup` — cleans up cached downloads (with `--outdated`, only cleans old downloads)
 * `home` — opens the homepage of the given Cask; or with no arguments, the homebrew-cask project page
 * `update` — a synonym for `brew update`
-* `zap` — try to remove *all* files associated with a Cask (including resources which may be shared with other applications)
+* `zap` — try to remove *all* files associated with a Cask (may include any resources shared with other applications)
```
Comment (vitorgalvao) [inline USAGE.md]: “any” seems unnecessary.

File: lib/hbc/cli/fetch.rb [hunk 4]
```diff
@@ -14,6 +14,6 @@ def self.run(*args)
   end
 
   def self.help
-    "downloads Cask resources to local cache"
+    "downloads Cask application files to local cache"
```
Comment (vitorgalvao) [inline lib/hbc/cli/fetch.rb]: “downloads remote application files to local cache”

File: doc/src/brew-cask.1.md (hunk 2 above)
Comment (Amorymeltzer) [inline doc/src/brew-cask.1.md]: Meant it in an "everything is files" sort of way; in the end, `uninstall` is just removing files...

File: lib/hbc/cli/fetch.rb (hunk 4 above)
Comment (Amorymeltzer) [inline lib/hbc/cli/fetch.rb]: :+1:

File: doc/src/brew-cask.1.md (hunk 2 above)
Comment (vitorgalvao) [inline doc/src/brew-cask.1.md]: But in that case, “files and resources” (your change in `CASK_LANGUAGE_REFERENCE.md`) stops making sense. I usually prefer “paths” to mean both files and directories, but I’m fine with just “files”, as long as we’re consistent.

Comment (Amorymeltzer) [inline doc/src/brew-cask.1.md]: Okay I see your point there, that's good to me.

**Pull request merged**